src/linear_algebra/clifford_algebra/conjugation.lean

```diff
@@ -40,7 +40,10 @@ section involute
 
 /-- Grade involution, inverting the sign of each basis vector. -/
 def involute : clifford_algebra Q →ₐ[R] clifford_algebra Q :=
-clifford_algebra.lift Q ⟨-(ι Q), λ m, by simp⟩
+begin
+  refine clifford_algebra.lift Q _,
+  exact ⟨-(ι Q), λ m, by simp only [linear_map.neg_apply, mul_neg, neg_mul, ι_sq_scalar, neg_neg]⟩
+end
```

Comment:
Did squeezing the simp not work without the separate `refine`, or is this just a stylistic choice?

Reply:
It didn't work without the separate `refine` when I removed the `irreducible` tag on `clifford_algebra` (I don't see how to make that irreducible in Lean 4). But since this is a can of worms, I have removed the Cliffod algebra stuff from this PR.

Reply:
Thanks for trying!